src/component/filterDimensionViewer.js

```diff
@@ -0,0 +1,60 @@
+import { toSparql } from 'clownface-shacl-path'
+import clownface from 'clownface'
+import $rdf from '@rdfjs/dataset'
+import { rdf } from '@tpluscode/rdf-ns-builders'
+import { ssz, view } from '../ns.js'
+
+export default {
```

Comment:
The viewer is hidden by default but it in the background (the `init` function) it ensures that a correct filter dimension is created, specifically one that combines the properties to build a property path as in the case of `<ZEIT>/<hasEnd>`